sections/main-product.liquid

```diff
@@ -1,740 +1,115 @@
-<section
-  id="MainProduct-{{ section.id }}"
-  class="page-width section-{{ section.id }}-padding"
-  data-section="{{ section.id }}"
->
-  {{ 'section-main-product.css' | asset_url | stylesheet_tag }}
-  {{ 'component-accordion.css' | asset_url | stylesheet_tag }}
-  {{ 'component-price.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rte.css' | asset_url | stylesheet_tag }}
-  {{ 'component-slider.css' | asset_url | stylesheet_tag }}
-  {{ 'component-rating.css' | asset_url | stylesheet_tag }}
-  {{ 'component-loading-overlay.css' | asset_url | stylesheet_tag }}
-  {{ 'component-deferred-media.css' | asset_url | stylesheet_tag }}
+<div class="isolate color-{{ section.settings.color_scheme }}">
```

Comment:
Maybe let's apply this class ('isolate') instead on the element that also has the class `product__media-wrapper` applied to it. Since this is what is being impacted by the shadows. Rather than applying it to the wrapper of the whole file. 👍 

I think it would be better so it's applied specifically where needed. 🤔